alonzo/test/test/Test/Cardano/Ledger/Alonzo/Trials.hs

```diff
@@ -216,18 +262,26 @@ go n =
       ( localOption
           -- (QuickCheckReplay (Just 117392)) -- after 206 tests
           (QuickCheckReplay (Just n))
-          (testProperty "preserves ADA" $ adaPreservationChain @(AllegraEra TestCrypto))
-          -- (propertyTests  @(AlonzoEra TestCrypto))
+          -- (testProperty "preserves ADA" $ adaPreservationChain @(AllegraEra TestCrypto))
+          (propertyTests @(AlonzoEra TestCrypto))
           -- (testProperty "Delegation Properties" (delegProperties @(AlonzoEra TestCrypto)))
       )
 
 go2 :: IO ()
 go2 =
   defaultMain
     ( localOption
-        -- (QuickCheckReplay (Just 117392)) -- after 206 tests for (OutputTooSmallUTxO ...) error
-        (QuickCheckReplay (Just 88)) -- after 7 tests for (FeeTooSmallUTxO (Coin 2530801) (Coin (-12)))
-        (testProperty "preserves ADA" $ adaPreservationChain @(AllegraEra TestCrypto))
+        (QuickCheckReplay (Just 213590))
+        (testProperty "Chain and Ledger traces cover the relevant cases" (withMaxSuccess 50 (relevantCasesAreCovered @(AlonzoEra TestCrypto))))
+    )
+
+go3 :: Int -> IO ()
+go3 n =
```

Comment:
It looks like go2 and go3 were used for exploration?

Reply:
yes. I will clean this up.